h/static/scripts/annotator/plugin/textselection.coffee

```diff
@@ -6,13 +6,17 @@ class Annotator.Plugin.TextSelection extends Annotator.Plugin
     @$ = Annotator.$
 
     # Register the event handlers required for creating a selection
-    $(document).bind({
-      "mouseup": @checkForEndSelection
-    })
+    if /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)
```

Comment:
We should probably be looking to detect touch devices by their capabilities, not their names. You might want to look at [how Modernizr currently feature detects touch devices](https://github.com/Modernizr/Modernizr/blob/master/feature-detects/touchevents.js).